python/private/pypi/extension.bzl

```diff
@@ -320,6 +331,16 @@ def _create_whl_repos(
                 ))
 
             whl_libraries[repo_name] = repo.args
+            if "experimental_target_platforms" in repo.args:
+                whl_libraries[repo_name] |= {
+                    "experimental_target_platforms": sorted({
+                        # TODO @aignas 2025-07-07: this should be solved in a better way
+                        platforms[candidate].tripple.partition("_")[-1]: None
```

Comment:
Yeah, isn't this problematic if platform names are user-chosen?

Reply:
This is constructed in a way that does not include the user-chosen platform names.